examples/textbooks/readme.md

```diff
@@ -1,5 +1,6 @@
-# Installation
+# Guidelines
 
-Install inference-swarm and the requirements in the example.
 
-Install `datatrove` from source
+```
+python ./examples/textbooks/generate_openstax.py --tgi_instances 2 --max_samples 10000
+```
```

Comment:
Maybe in the readme do some screenshot of what it's supposed to look like. E.g., see the readme.

<img width="959" alt="image" src="https://github.com/huggingface/llm-swarm/assets/5555347/4ea48fef-fee6-4186-bbbf-29df797e9118">

Reply:
added it